src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -1,8 +1,9 @@
-import React, { useState } from "react";
+import React, { useState, useCallback } from "react";
 
-import { DEFAULT_USER_LIST_LIMIT } from "settings";
+import { currentVenueSelectorData, parentVenueSelector } from "utils/selectors";
+import { openUrl, venueInsideUrl } from "utils/url";
 
-import { currentVenueSelectorData } from "utils/selectors";
+import { DEFAULT_USER_LIST_LIMIT } from "settings";
```

Comment:
Settings should be right after the 3rd party lib imports

Reply:
fixed in fae8270fb609124c5b2b41895cad7dced8bf0718